pkg/kubelet/cm/deviceplugin/kubelet_internal_checkpoint

```diff
@@ -0,0 +1 @@
+{"PodDeviceEntries":null,"RegisteredDevices":{"domain1.com/resource1":["dev1","dev2"],"domain2.com/resource2":["dev3","dev4"]}}
```

Comment:
What's this file for? Seems weird.

Reply:
Ah ... my fault. It is a temp file generated during unit tests. Will remove.
Thanks for the review.